server/routes/api/documents/documents.ts

```diff
@@ -1701,4 +1701,48 @@ router.post(
   }
 );
 
+router.post(
+  "documents.empty_trash",
+  auth({ admin: true }),
+  validate(T.DocumentsEmptyTrashSchema),
+  async (ctx: APIContext<T.DocumentsEmptyTrashReq>) => {
+    const { user } = ctx.state.auth;
+    const { ids } = ctx.input.body;
```

Comment:
We don't need to send `ids`, it's not particularly likely that the client will have the ids of every document in the trash in fact. Better to have the server find deleted docs that the user has access to

Reply:
Oh yeah, pagination.

So, the API would need to send the deleted ids back to the UI (to remove those docs from the store). Otherwise, the UI has to refresh all collections & docs to fetch the latest state (which seems un-necessary).
Is this an acceptable approach?

Reply:
I'm not sure if it needs to send the ids back as the client can just remove any deleted documents it has?